tests/MigrationTest.php

```diff
@@ -64,18 +64,18 @@ public function testCreateMethod()
 
             $this->assertTrue(true, 'Migration ok');
         } catch (\Exception $e) {
-            $this->assertTrue(false, $e->getMessage());
+            $this->assertFalse(false, $e->getMessage());
```

Comment:
Consider following `assertTrue` and `assertFalse` assertions:

- The `assertTrue` is about `assertTrue($result, 'description message')`.
- The `assertFalse` is about `assertFalse($result, 'description message')`.

That is, the `assertFalse` will assert the `$result` is expected to `false`, using the `assertFalse` instead.